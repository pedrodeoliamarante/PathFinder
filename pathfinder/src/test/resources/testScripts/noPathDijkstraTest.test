# Find a path with Dijkstra's algorithm when
# the paths tested don't exist

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B

AddEdge simpleGraph A B 10.0

FindPath simpleGraph A C

FindPath simpleGraph C B