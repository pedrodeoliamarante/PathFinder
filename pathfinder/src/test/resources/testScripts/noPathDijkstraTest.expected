# Find a path with Dijkstra's algorithm when
# the paths tested don't exist

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph

added edge 10.000 from A to B in simpleGraph


unknown: C
unknown: B